README.rst

```diff
@@ -285,26 +285,29 @@ following these steps:
    need 2 new IP addresses in every region and both security groups
    need to be updated to include a total of 4 new addresses.
 
+#. Choose a private IP for the new instance, that is not already taken by any
+   other EC2 instance in the VPC.  You will need it on further steps.
+
+#. Create a new EBS volume of appropriate type and size (normally you want to
+   have the same settings as for the rest of the cluster).  EBS encryption is
+   not recommended as it might prevent auto-recovery.
+
+#. Create a ``Name`` tag on the volume in the format:
+   ``<cluster-name>-<private-ip>``.
+
 #. Use the 'Launch More Like This' menu in the AWS web console on one
    of the running nodes.
 
-#. If present remove the volumes section from user data. It references
-   the data volume of the original instance.
-
```

Comment:
The data volume in the user data must be updated to match the IP of the new node

Reply:
Mentioned below: https://github.com/zalando-stups/planb-cassandra/pull/187/files#diff-88b99bb28683bd5b7e3a204826ead112R308

Reply:
True, I missed it. Thanks!